core/components/molecules/emptyState/__stories__/index.story.jsx

```diff
@@ -100,7 +100,11 @@ export const all = () => {
   );
 };
 
-const customCode = `() => {
+const customCode = `
```

Comment:
@anuradha9712  Please update all the imports to use the camel case variable names. For example 
`import noFilesEmptyFiles from '@innovaccer/mds-images/ui-states/no-files-empty-files.svg';`
in stead of `import no-files-empty-files from '@innovaccer/mds-images/ui-states/no-files-empty-files.svg';`
"-" Hyphen is not allowed in variable names.